bin/htmldiff

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+#
+# This file was generated by Bundler.
+#
+# The application 'htmldiff' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require "pathname"
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
+  Pathname.new(__FILE__).realpath)
+
+require "rubygems"
+require "bundler/setup"
+
+load Gem.bin_path("diff-lcs", "htmldiff")
```

Comment:
Looks like a tool for seeing diffs in html file. It's come with the bundle installation:

> # This file was generated by Bundler.
> #
> # The application 'htmldiff' is installed as part of a gem, and
> # this file is here to facilitate running it.